src/webapp/github.py

```diff
@@ -76,7 +136,10 @@ def publish_pr_review(self, owner, repo, num, body, action, sha):
         # action: APPROVE, REQUEST_CHANGES, or COMMENT
         api_path = "/repos/:owner/:repo/pulls/:number/reviews"
         url = api_path2url(api_path, owner=owner, repo=repo, number=num)
-        data = {'event': action, 'commit_id': sha}
+        data = {
+            'event': action,
+            'commit_id': sha
+        }
```

Comment:
Did your editor do this for you while you weren't looking?

Reply:
I probably did it well I wasn't looking, I sometimes mindlessly format when I am thinking.